interaction/templates/interaction/interactionselection.html

```diff
@@ -54,30 +64,41 @@
 
             <div class="col-md-12 panel panel-primary">
                 <div class="panel-body">
-                    <h4>Select from interactions pre-calculated</h4>
+                    <h4>Structure select</h4>
                     <div>
-                    <select id='preloaded_xtals'>
-                    <option>Select from list</option>
-                    {% for structure in structures %}
-                        <option value="{{structure.structure_ligand_pair__structure__pdb_code__index}}">{{structure.structure_ligand_pair__structure__pdb_code__index}} ({{structure.structure_ligand_pair__structure__protein_conformation__protein__parent__entry_name}}) </option>
-                    {% endfor %}
-                    </select>
+                      <div class="dropdown" id='preloaded_xtals'>
+                        <button class="btn btn-default dropdown-toggle" type="button" data-toggle="dropdown">Select from list<span class="caret"></span></button>
+                        <ul class="dropdown-menu" role="menu">
+                          {% for group in structure_groups %}
+                            <li class="dropdown-submenu">
+                              <a tabindex="-1" href="#">PDB-code: {{ group }}xxx</a>
+                              <ul class="dropdown-menu structure_select">
+                                {% for structure in structures %}
+                                  {%  if structure.structure_ligand_pair__structure__pdb_code__index|first == group %}
+                                    <li><a href="#" onClick="selectedPDBAction('{{structure.structure_ligand_pair__structure__pdb_code__index}}');">{{structure.structure_ligand_pair__structure__pdb_code__index}} ({{structure.structure_ligand_pair__structure__protein_conformation__protein__parent__entry_name}})</a></li>
+                                  {% endif %}
+                                {% endfor %}
+                              </ul>
+                            </li>
+                          {% endfor %}
+                        </ul>
+                      </div>
                     </div>
                 </div>
             </div>
 
-            <div class="col-md-12 panel panel-primary">
+<!--            <div class="col-md-12 panel panel-primary">
```

Comment:
Maybe can be changed to {# #}, instead of <!-- -->?

Reply:
Done!